feature/mypage/src/main/java/com/mument_android/mypage/MyPageActivity.kt

```diff
@@ -76,11 +82,29 @@ class MyPageActivity : BaseActivity<ActivityMyPageBinding>(R.layout.activity_my_
         startActivity(intent)
     }
 
+    private fun logoutBtnListener() {
+        binding.tvMyPageLogout.setOnClickListener {
+            MumentDialogBuilder()
+                .setHeader(getString(R.string.logout_header))
+                .setBody("")
+                .setOption(MumentDialog.DIALOG_LOGOUT)
+                .setAllowListener {
+                    finish()
+                    val intent = Intent(this, LogInActivity::class.java)
+                    startActivity(intent)
```

Comment:
- 이 방식이 더 좋을 것 같네요~!!
```suggestion
                    val intent = Intent(this, LogInActivity::class.java)
                    startActivity(intent)
                    finish()
```

Reply:
넵 수정하겠습니다!!

Reply:
Commit Suggestion 해주셔도 좋숩니당~!